features/steps/core.py

```diff
@@ -6,11 +6,18 @@
 from jrnl import __version__
 from dateutil import parser as date_parser
 from collections import defaultdict
+try: import parsedatetime.parsedatetime_consts as pdt
+except ImportError: import parsedatetime as pdt
+import time
 import os
 import json
 import yaml
 import keyring
 
+consts = pdt.Constants(usePyICU=False)
+consts.DOWParseStyle = -1 # Prefers past weekdays
+CALENDAR = pdt.Calendar(consts)
+
```

Comment:
Did we mean to put this back? This was removed in earlier work. Is there a reason this needs to go back?

Reply:
So, I was using a phrase in core.py `check_output_time_inline` for a couple of my scenarios testing the `-on` command line option on a certain date. When I searched the code, it wasn't being used anywhere. 

Doing a little digging, it _had_ been used in the dayone.feature file for testing entries that didn't have timezone information (i think). However all those tests have been removed.
 
The function didn't work for my needs because it didn't handle day names (i.e. 'monday', etc.) So I called the parsedatetime library instead of the date parser.

That is why I put the code in. All the tests passed, so I didn't look any further. :-)